Sources/MapboxMaps/Foundation/Tracing.swift

```diff
@@ -0,0 +1,98 @@
+import Foundation
+import os.signpost
```

Comment:
Are there any benefits in that form of imports? 
The last time I investigated there were nothing so I would prefer to keep simple `import os`